ArduCopter/AP_Arming.cpp

```diff
@@ -390,8 +390,13 @@ bool AP_Arming_Copter::gps_checks(bool display_failure)
     fence_requires_gps = (copter.fence.get_enabled_fences() & (AC_FENCE_TYPE_CIRCLE | AC_FENCE_TYPE_POLYGON)) > 0;
     #endif
 
+    bool check_gps_all_modes = false;
+    #if ARM_GPS_ALL_MODES == ENABLED
+    arm_gps_all_modes = true;
```

Comment:
Should this variable be `check_gps_all_modes`

Reply:
Thanks, I missed that. Fixed.